lib/mavenlink/workspace_resource.rb

```diff
@@ -1,4 +1,7 @@
 module Mavenlink
   class WorkspaceResource < Model
+    def allocations_matching_scheduled_hours(hard = false, occurrence = {})
+      client.post("#{collection_name}/#{@id}/allocations_matching_scheduled_hours", hard: hard, occurrence: occurrence)
```

Comment:
Does this POST response return anything that we should potentially return here?

Reply:
No, on postman it was an empty hash {}